pkg/kubelet/images/image_manager.go

```diff
@@ -189,19 +190,18 @@ func evalCRIPullErr(container *v1.Container, err error) (errMsg string, errRes e
 // applyDefaultImageTag parses a docker image string, if it doesn't contain any tag or digest,
 // a default tag will be applied.
 func applyDefaultImageTag(image string) (string, error) {
-	named, err := dockerref.ParseNormalizedNamed(image)
+	_, tag, digest, err := parsers.ParseImageName(image)
 	if err != nil {
 		return "", fmt.Errorf("couldn't parse image reference %q: %v", image, err)
```

Comment:
return this error as-is so we don't double-wrap:
```suggestion
		return "", err
```

and update `ParseImageName` to include the image name that failed parsing:
```go
return "", "", "", fmt.Errorf("couldn't parse image name %q: %v", image, err)
```

Reply:
done.